swift/IceStorm/clock/README.md

```diff
@@ -1,168 +1,131 @@
-Overview
-========
+# IceStorm Clock
+
+## Overview
 
 This demo shows how to use [IceStorm][1].
 
 To run the demo, start the IceStorm service:
 
-```
+```shell
 icebox --Ice.Config=config.icebox
 ```
 
-This configuration assumes there is a subdirectory named db in the
-current working directory.
+This configuration assumes there is a subdirectory named db in the current working directory.
 
 In a separate window:
 
-```
-./Build/subscriber
+```shell
+swift run Subscriber
 ```
 
 In another window:
 
-```
-./Build/publisher
+```shell
+swift run Publisher
 ```
 
-While the publisher continues to run, "tick" messages should be
-displayed in the subscriber window.
+While the publisher continues to run, "tick" messages should be displayed in the subscriber window.
 
-Options
-=======
+## Options
 
-Both the subscriber and publisher take an optional topic name as a
-final argument. The default value for this topic is "time".
+Both the subscriber and publisher take an optional topic name as a final argument. The default value for this topic
+is "time".
 
-Through the use of command-line options, both the subscriber and
-publisher can use different QoS for sending and receiving messages.
+Through the use of command-line options, both the subscriber and publisher can use different QoS for sending and
+receiving messages.
 
-Subscriber Options
-------------------
+## Subscriber Options
 
-```
-./Build/subscriber --oneway
+```shell
+swift run Subscriber --oneway
 ```
 
-The subscriber receives events as oneway messages. This is the
-default.
+The subscriber receives events as oneway messages. This is the default.
 
-```
-./Build/subscriber --datagram
+```shell
+swift run Subscriber --datagram
 ```
 
 The subscriber receives events as datagrams.
 
-```
-./Build/subscriber --twoway
+```shell
+swift run Subscriber --twoway
 ```
 
 The subscriber receives events as twoway messages.
 
-```
-./Build/subscriber --ordered
+```shell
+swift run Subscriber --ordered
 ```
 
-The subscriber receives events as twoway messages with guaranteed
-ordering.
+The subscriber receives events as twoway messages with guaranteed ordering.
 
-```
-./Build/subscriber --batch
+```shell
+swift run Subscriber --batch
 ```
 
-This is an additional flag that forwards datagram and oneway events
-to the subscriber in batches.
+This is an additional flag that forwards datagram and oneway events to the subscriber in batches.
 
-```
-./Build/subscriber --id <id>
+```shell
+swift run Subscriber --id <id>
 ```
 
-This option specifies a unique identity for this subscriber. When
-you use this option, you should also run the subscriber on a fixed
-port by setting the `Clock.Subscriber.Endpoints` property. For
-example:
+This option specifies a unique identity for this subscriber. When you use this option, you should also run the
+subscriber on a fixed port by setting the `Clock.Subscriber.Endpoints` property. For example:
```

Comment:
That's why we parse with the Clock prefix in the Subscriber. That's code for advanced users!